src/cms/templates/authentication/activation_email_html.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n %}
+{% autoescape off %}
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+</head>
+<body>
+    <div style="text-align:justify; font-family:sans-serif; background-color: rgba(241, 245, 249, 100); padding: 1rem; border-radius: 1rem; line-height:1.5">
+        <img src='cid:logo' alt="" style="width:200px;"> 
+        <br>
+        {% trans 'Welcome' %} {{ user.profile.full_user_name }}, 
+        <br> 
+        <br>    
+        {% trans 'You are receiving this e-mail, because an Integreat-account was created for you.' %}
+        {% trans 'Please click the link below to set a password and activate your account.' %}
+        <br>
+        <br>
+        <a href={{ base_url }} style="color:rgb(0, 0, 200);"> {{ base_url }}{% url 'activate_account' uidb64=uid token=token %} </a>
+        <br>
+        <br>
+        {% trans 'If clicking on the link above does not work, please copy the URL and paste it into the address bar of your browser.' %}
+        <br>
+        {% trans 'Your username is:' %} <span style="font-weight:bold"> {{ user.get_username }} </span> 
+        <br>
+        <br>
+        {% trans 'Best regards' %}, 
+        <br>
+        {% trans 'Your Integreat team' %}{% if region %} {{ region.name }}{% endif %}
+    </div>
+</body>
+{% endautoescape %}
```

Comment:
Since you're now only using the `welcome_email.html` template, you can delete this file.